Description: Replace 'locale -a' output with accountsservice's list of language options.
Forwarded: not-needed
Author: Gunnar Hjalmarsson <ubuntu@gunnar.cc>

Index: lightdm-1.5.1/liblightdm-gobject/language.c
===================================================================
--- lightdm-1.5.1.orig/liblightdm-gobject/language.c	2013-03-08 11:18:25.733614639 +1300
+++ lightdm-1.5.1/liblightdm-gobject/language.c	2013-03-08 11:18:25.733614639 +1300
@@ -40,7 +40,7 @@
 static void
 update_languages (void)
 {
-    gchar *command = "locale -a";
+    gchar *command = "/usr/share/language-tools/language-options";
     gchar *stdout_text = NULL, *stderr_text = NULL;
     gint exit_status;
     gboolean result;
